doc/user/content/sql/system-catalog.md

```diff
@@ -249,6 +249,19 @@ Field          | Type       | Meaning
 `schema_id`    | [`bigint`] | The ID of the schema to which the sink belongs.
 `name`         | [`text`]   | The name of the sink.
 
+### `mz_source_info`
+
+The `mz_source_info` table contains a row for each source in the system.
+
+Field          | Type       | Meaning
+---------------|------------|----------
+`source_name`  | [`text`]   | Materialize's internal name for the source.
+`source_id`    | [`text`]   | Materialize's unique ID for the source. Equivalent to `mz_sources.id`.
```

Comment:
I know this is just matching the text in `mz_kafka_consumer_stats`, but "equivalent to" doesn't sound quite right to my ear. I propose that we say "References `mz_sources.id`" instead. (And similarly in `mz_kafka_consumer_stats`). That's the language that PostgreSQL uses.

Reply:
Thanks for pointing this out! When Postgres uses the word "references", does that imply a foreign key relationship? I chose the word "equivalent" because we don't maintain any explicit relationship / reference between the values, nor do we enforce that a row exists in `mz_sources` for each `mz_kafka_consumer_stats.source_id`.

Reply:
Yeah, it implies a foreign key relationship. I still think it is more correct for us to say "references" rather than "equivalent to", even though we don't quite enforce the foreign key relationship. I just don't think that "column t1`.a is equivalent to t2.id" means the right thing. To me the "equivalence" framing implies that the columns are literally the same, not that the values in the former reference values in the later.

What about "corresponds" as a term that neither implies a foreign key relation, but also does not imply an equivalence relation.

Reply:
Sounds good to me! I'll update it now.